infra-templates/pipeline/config.yml

```diff
@@ -0,0 +1,7 @@
+name: Rancher Pipeline 
+description: (Experimental) Rancher CI service
```

Comment:
No need to put in Experimental in the name a we have the experimental in the banner.

Reply:
I'll remove that.